tests/testInstanceCatalog.py

```diff
@@ -126,6 +126,14 @@ class unicodeCannotBeNullCatalog(InstanceCatalog):
     cannot_be_null = ['n5']
 
 
+class severalCannotBeNullCatalog(InstanceCatalog):
+    """
+    This catalog class will not write rows with a null value in the n2 column
```

Comment:
I also think the doc strings for the cannotBeNull Catalogs above must be corrected in the same way.